specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -499,15 +594,680 @@
           "description": "The entity confidence score.",
           "type": "number"
         },
-        "listKeys": {
+        "resolutions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BaseResolution"
+          }
+        },
+        "extraInformation": {
           "type": "array",
-          "description": "List of keys.",
+          "items": {
+            "$ref": "#/definitions/BaseExtraInformation"
+          }
+        }
+      }
+    },
```

Comment:
`Schema should have a description or title.` 
There are a few places in the open api document where this information should be added. The descriptions will be part of the automatically generated [developer documentation](https://review.docs.microsoft.com/en-us/rest/api/azure-rest-preview/?branch=result_c629252e-89a7-4b54-a606-d3f7d13e2f9d&view=azure-rest-preview).

Reply:
Thanks Mark, the missing descriptions were all added.